### PR TITLE
Add error handler description to README

### LINKED ISSUE #187: fetch_url_error: This URL returned a 524 status code
Thanks for the great library, I've been using it for years and after upgrading to v3.1.0 I'm running into problems so I had to revert to using v2.0.1

This is the trace that I keep seeing:
```events.js:182
      throw er; // Unhandled 'error' event
      ^

fetch_url_error: This URL returned a 524 status code
    at Request.request.get.on (/code/rssalerts/node_modules/rss-feed-emitter/src/Feed.js:128:28)
    at emitOne (events.js:115:13)
    at Request.emit (events.js:210:7)
    at Request.onRequestResponse (/code/rssalerts/node_modules/request/request.js:1066:10)
    at emitOne (events.js:115:13)
    at ClientRequest.emit (events.js:210:7)
    at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:565:21)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:116:23)
    at Socket.socketOnData (_http_client.js:454:20)
    at emitOne (events.js:115:13)
    at Socket.emit (events.js:210:7)
    at addChunk (_stream_readable.js:266:12)
    at readableAddChunk (_stream_readable.js:253:11)
    at Socket.Readable.push (_stream_readable.js:211:10)
    at TCP.onread (net.js:587:20)
```

### PR DESCRIPTION
**Pull request by alek-p**
This documents the best practice of adding an error event handler.

closes #187

### CODE REVIEW

**Comment by TobiTenno** [comment]
This is already part of the [node.js documentation for Events](https://nodejs.org/api/events.html#events_error_events)

File: README.md [hunk 1]
```diff
@@ -146,6 +146,12 @@ feeder.on('nintendo', function(item) {
 });
 ```
 
+### Adding an 'error' handler
+Handle error events by printing to console. This handler is needed to prevent unhandled exceptions from crashing the processes.
+```js
+feeder.on('error', (err) => {console.error(err)});
+```
```
Comment (TobiTenno) [inline README.md:153]:
```suggestion
```js
feeder.on('error', console.error);
```
```

**Comment by alek-p** [comment]
> This is already part of the [node.js documentation for Events](https://nodejs.org/api/events.html#events_error_events)

I didn't know about it, so adding to the README could help people

**Pull request merged**